app/grants/urls.py

```diff
@@ -20,17 +20,18 @@
 from django.urls import path, re_path
 
 from grants.views import (
-    add_grant_from_collection, api_toggle_user_sybil, bulk_fund, bulk_grants_for_cart, cancel_grant_v1, cart_thumbnail,
-    clr_grants, collage, collection_thumbnail, contribute_to_grants_v1, contribution_addr_from_all_as_json,
-    contribution_addr_from_grant_as_json, contribution_addr_from_grant_during_round_as_json,
-    contribution_addr_from_round_as_json, contribution_info_from_grant_during_round_as_json, create_matching_pledge_v1,
-    delete_collection, flag, get_clr_sybil_input, get_collection, get_collections_list, get_ethereum_cart_data,
-    get_grant_payload, get_grant_tags, get_grants, get_interrupted_contributions, get_replaced_tx, get_trust_bonus,
-    grant_activity, grant_details, grant_details_api, grant_details_contributions, grant_details_contributors,
-    grant_edit, grant_fund, grant_new, grants, grants_addr_as_json, grants_bulk_add, grants_by_grant_type,
-    grants_cart_view, grants_info, grants_landing, grants_type_redirect, hall_of_fame, ingest_contributions,
-    ingest_contributions_view, invoice, leaderboard, manage_ethereum_cart_data, new_matching_partner, profile,
-    quickstart, remove_grant_from_collection, save_collection, toggle_grant_favorite, upload_sybil_csv, verify_grant,
+    GrantSubmissionView, add_grant_from_collection, api_toggle_user_sybil, bulk_fund, bulk_grants_for_cart,
```

Comment:
the `GantSbumissionView` is new - the rest of the changes in this are linting fixes